webpack/InsightsCloudSync/InsightsCloudSyncConstants.js

```diff
@@ -3,13 +3,15 @@ import { getControllerSearchProps } from 'foremanReact/constants';
 import { foremanUrl } from '../ForemanRhCloudHelpers';
 
 export const INSIGHTS_CLOUD_SYNC_SUCCESS = 'INSIGHTS_CLOUD_SYNC_SUCCESS';
+
 export const INSIGHTS_SYNC_PAGE_TITLE = __('Red Hat Insights');
+
 export const INSIGHTS_PATH = foremanUrl('/foreman_rh_cloud/insights_cloud');
-export const INSIGHTS_HITS_PATH = '/insights_cloud/hits';
+
 export const INSIGHTS_SAVE_AND_SYNC_PATH = foremanUrl(
   '/insights_cloud/save_token_and_sync'
 );
-export const INSIGHTS_HITS_API_KEY = 'INSIGHTS_HITS';
```

Comment:
This is used in `webpack/InsightsCloudSync/Components/NoTokenEmptyState.js`

Reply:
good catch!